x/gov/types/keys.go

```diff
@@ -50,10 +52,10 @@ var (
 	VotesKeyPrefix = []byte{0x20}
 
 	// ParamsKey is the key to query all gov params
-	ParamsKey = []byte{0x30}
+	ParamsKey = collections.NewPrefix(30)
 
 	// KeyConstitution is the key string used to store the chain's constitution
-	KeyConstitution = []byte("constitution")
+	KeyConstitution = collections.NewPrefix("constitution")
```

Comment:
Can we rename this variable as well, this never have been released, so we still can.